content/blog/2021/08/2021-02-08.adoc

```diff
@@ -0,0 +1,167 @@
+
+We can see how easily that can grow in cost and maintenance. A preferred way is to have a common language each of these services use and understand as a way to communicate amongst each other. This way, an event which is emitted using this common language will be available to any of the interested receiver without that receiver needing a special agent. This way of communicating using a common/standard language also increases agnostic communication where the sender or the receiver are sending and receiving data without creating a tight coupling between the two. 
+
+CloudEvents specification is enabling that loosely-coupled, event-driven communication between services by enforcing that common language for how an event should be emitted and transferred. 
+
+
+== CloudEvents and Jenkins
+
+image:https://cncf-branding.netlify.app/img/projects/cloudevents/stacked/color/cloudevents-stacked-color.png[CloudEvents Specification, link=https://cloudevents.io/, align="center"]
+
+  A specification for describing event data in a common way
+
+____
+* Consistency
+    - Consistent across tools and services. 
+* Accessibility
+    - Common event format means common libraries, tooling, and infrastructure for delivering event data across environments can be used to develop with CloudEvents. 
+* Portability
+    - Easily port event-data across tools, truly leveraging event-driven architecture.
+____
+
+The CloudEvents plugin for Jenkins is developed as an effort to make interoperability between Jenkins and CI/CD tools much easier. The CloudEvents plugin for Jenkins is a GSoC project, and with the help from an amazing team of mentors, this project is aimed at enhancing event-driven interoperability between cloud-native CI/CD tools, making it easier for developers to include jenkins in their CI/CD pipelines.
+
+With this plugin, Jenkins can send and receive CloudEvents-compliant events to and from a wide variety of CI/CD tools using CloudEvents as their event format. This plugin makes chaining Jenkins with multiple tools like Tekton, Keptn, Knative and more, very easy. 
+
+
+== GSoC Phase 1 - CloudEvents Plugin
+
+=== Using CloudEvents plugin for Jenkins
+
+This plugin allows Jenkins to be configured as a source and sink, which can emit and consume CloudEvents from a range of tools simultaneously. 
+
+==== Jenkins as a Source
+Configuring Jenkins as a Source enables Jenkins to send CloudEvents to a CloudEvents sink. For Phase-I of this project, there is support for HTTP Sinks, however CloudEvents supports various link:https://github.com/cloudevents/spec#cloudevents-documents[protocol bindings]. Moving forward, there will also be support for other protocol bindings supported by CloudEvents. 
+
+
+To use Jenkins as a Source, the following configuration is needed:
+
+1. Click on *Manage Jenkins* in the Root-Actions menu on the left.
+image:https://github.com/ShrutiC-git/cloudevents-plugin/blob/main/public/manage_jenkins.png[Manage Jenkins]
```

Comment:
This image does not work. IMO the picture here is optional but if you really want one it's best to capture it and upload it as an image.